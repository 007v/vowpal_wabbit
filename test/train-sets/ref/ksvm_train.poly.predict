0
0.473179
0.043690
-0.214710
-0.445576
-0.480717
-0.202344
-0.193857
-0.462571
-0.561376
-0.512644
-0.385758
-0.461433
-0.538895
-0.445232
-0.287877
-0.409284
-0.266725
-0.404348
-0.386682
-0.507182
-0.478388
-0.274970
0.045114
-0.019228
-0.068550
-0.212493
-0.690429
-0.320477
-0.235376
-0.160128
-0.115747
-0.305755
-0.204343
-0.192181
-0.169748
-0.260945
-0.103221
-0.158711
-0.123812
-0.390752
-0.232305
-0.218394
-0.186186
-0.355086
-0.581495
-0.087426
-0.093389
-0.532301
-0.181551
-0.375556
-0.337506
-0.387462
-0.027746
-0.284967
0.247813
-0.169308
-0.265988
-0.140870
-0.366559
-0.179408
-0.029357
0.278270
-0.642497
-0.061667
-0.128920
-0.120760
-0.141874
-0.435964
-0.062886
0.145994
0.176412
-0.172625
-0.455073
0.271291
0.097244
-0.015747
-0.221408
-0.243711
-0.289540
-0.114238
-0.276432
0.074297
-0.138327
-0.407983
-0.454601
-0.518856
0.107507
-0.158953
-0.781124
-0.324744
-0.265495
0.494075
0.000198
0.397777
-0.698875
-0.338643
-0.100292
-0.329382
-0.107264
-0.050456
-0.223781
0.028869
-0.105229
-0.045446
-0.069673
-0.484189
-0.272836
-0.349343
-0.066151
0.131945
-0.051699
-0.235854
-0.368971
0.212958
-0.423758
-0.468738
-0.068814
-0.263291
-0.232410
-0.242514
-0.076915
-0.274164
-0.530904
0.017772
-0.058512
-0.267377
0.077727
-0.667906
-0.095525
-0.244298
0.227481
-0.019186
-0.082720
-0.509806
-0.386706
0.004987
-0.100471
0.509179
-0.426297
-0.307782
-0.375925
0.604166
0.721092
-0.097181
0.260943
-0.136194
0.810269
-0.632994
-0.156846
0.068606
0.036682
0.214582
-0.148372
-0.443142
0.107640
0.433307
-0.134486
0.528804
-0.100775
0.967918
-0.543109
-0.438068
0.183014
-0.033117
0.056194
0.525197
-0.104783
-0.230966
0.649036
0.546901
-0.458022
-0.367629
0.194318
-0.459769
-0.028860
-0.667517
0.623121
0.106218
0.382619
-0.069243
-0.207739
0.294620
-0.448535
0.007227
-0.041096
0.352865
-0.214175
-0.225165
0.377092
0.476515
0.022086
-0.041722
-0.203140
-0.078421
0.009179
0.121588
-0.388073
-0.046198
0.047478
-0.223496
0.061431
0.081135
-0.360467
-0.221227
-0.407570
0.519418
0.110672
-0.278357
0.015986
0.031402
-0.586125
-0.415026
0.269010
1.039417
-0.692839
-0.340467
-0.160254
-0.104900
-0.167661
0.040728
-0.705058
-0.214507
-0.112547
0.157883
-0.165190
1.064672
0.467304
-0.235725
-0.443357
-0.624732
-0.440861
-0.648015
0.086313
0.203145
-0.654345
-0.197217
0.261814
0.731830
0.211699
0.500392
-0.097749
-0.713442
0.727595
0.020262
0.113346
0.588077
-0.217992
-0.442638
0.265731
