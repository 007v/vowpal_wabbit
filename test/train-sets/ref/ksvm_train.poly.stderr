using l2 regularization = 1
predictions = ksvm_train.poly.predict
Lambda = 1
Kernel = poly
degree = 2
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.236589 1.473179            2            2.0  -1.0000   0.4732      103
1.075540 0.914490            4            4.0  -1.0000  -0.2147      134
0.992637 0.909735            8            8.0  -1.0000  -0.1939      145
0.929643 0.866648           16           16.0   1.0000  -0.2879       23
0.938440 0.947237           32           32.0  -1.0000  -0.1157       31
0.938311 0.938183           64           64.0  -1.0000  -0.6425       60
0.926805 0.915298          128          128.0   1.0000   0.0777      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.831654
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 247
Number of kernel evaluations = 171073 Number of cache queries = 112947
Total loss = 207.913681
Done freeing model
Done freeing kernel params
Done with finish 
